### PR TITLE
117 standardizing navigation pages for page nav

### LINKED ISSUE #117: Standardizing navigation pages for page nav #110
Update pages on main navigation to split sections into separate pages. Keep each main section on it's own page.

**AC**
- [x] Split current sections into separate pages
- [x] Verify Navigation opens and closes per section
- [x] Direct links exist for each section with anchors for sub section

### PR DESCRIPTION
**Pull request by roboza**
Closes #117 
all pages are separated into their own pages

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -5,15 +5,9 @@ toc:
     position: 1
   - title: "Quickstart"
     collection: "quickstart"
-    url: "/"
+    url: "/quickstart.html"
```
Comment (relativityboy) [inline _data/nav.yml]: I'd like us to get rid of `.html` in the urls.

File: _data/nav.yml [hunk 2]
```diff
@@ -28,16 +22,18 @@ toc:
         collection: tutorials-ruby
   - title: "Services"
     collection: "services"
-    url: "/"
+    url: "/services.html"
```
Comment (relativityboy) [inline _data/nav.yml]: remove `.html`

File: _data/nav.yml [hunk 3]
```diff
@@ -28,16 +22,18 @@ toc:
...
+    exclude: true
   - title: "Appbase API"
     collection: apidefinitions    
     url: "/apidefinitions.html"
     exclude: true
     position: 7    
   - title: "Community"
     collection: "community"
-    url: "/"
+    url: "/community.html"
```
Comment (relativityboy) [inline _data/nav.yml]: remove `.html`

**Comment by roboza** [comment]
I have added permalinks. For whatever reason if I move the html files to the folders as index.html it does not render so have added the permalink to each of those html files. They still render properly and you now don't need to enter the .html in the url.

**Pull request merged**